docs/spec/staking/state.md

```diff
@@ -2,46 +2,46 @@
 
 ### Pool
 
-The pool is a space for all dynamic global state of the Cosmos Hub.  It tracks
-information about the total amounts of Atoms in all states, moving Atom
-inflation information, etc.
+The pool dynamically tracks information about the total amounts of staking
+tokens and their states (bonded or loose). Note that `NotBondedTokens` _does_
+include both `unbonding` tokens, and fully `unbonded` tokens. 
 
  - Pool: `0x01 -> amino(pool)`
 
 ```golang
 type Pool struct {
-    LooseTokens         sdk.Int   // tokens not associated with any bonded validator
-    BondedTokens        sdk.Int   // reserve of bonded tokens
+    NotBondedTokens sdk.Int   // tokens not associated with any bonded validator
+    BondedTokens    sdk.Int   // reserve of bonded tokens
 }
 ```
 
 ### Params
 
 Params is global data structure that stores system parameters and defines
-overall functioning of the staking module. 
+overall functioning of the staking module. it is stored 
```

Comment:
Maybe leave off `it is stored` here. Link to the paramstore spec ideally.

Reply:
yeah this is typo removing